test/conformance/kuttl/reports/background/verify-image-pass/README.md

```diff
@@ -0,0 +1,9 @@
+# Title
+
+This test create an image verification policy runing in the background.
+It then creates a pod that satisfies the policy.
+Note: the pod has to be created first because we don't want the policy to apply at admission time.
```

Comment:
```suggestion
This test creates a pod using a valid signed image.
It then creates an image verification policy running in the background.

Note: the pod has to be created first because we don't want the policy to apply at admission time.
```